### PR TITLE
90 same Ansible users everywhere

### PR DESCRIPTION
**Pull request by fdiblen**
Refs: #90

### CODE REVIEW

File: ubuntu-vagrant/Vagrantfile [hunk 1]
```diff
@@ -67,4 +67,7 @@ Vagrant.configure("2") do |config|
   #   apt-get update
   #   apt-get install -y apache2
   # SHELL
+
+  # set default ssh user
+  config.ssh.username = "tester"
```
Comment (sverhoeven) [inline ubuntu-vagrant/Vagrantfile:72]: This just changes the external username, not the user inside the VM. The vagrant box has a `vagrant` user and no `tester` user.  Doing a `vagrant up` fails with authentication failures with this change. 

You could add a provision script to create the tester user, but the user that vagrant uses should stay `vagrant`.

Comment (sverhoeven) [reply ubuntu-vagrant/Vagrantfile:72]: https://www.vagrantup.com/docs/provisioning/shell

File: ubuntu-surf-hpc-cloud/playbook.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 - name: GitHub Actions Runner
   hosts: hpchosts
-  # become: yes
+  become: yes
   vars:
     # you can pin the runner version otherwise the latest version will be used
     # - runner_version: "2.263.0"
-    - runner_user: runner
+    - runner_user: tester
```
Comment (jspaaks) [inline ubuntu-surf-hpc-cloud/playbook.yml:8]:
```suggestion
```

I think we can just have ``ansible_user`` below, no need to make a ``runner_user``. E.g. 

https://github.com/ci-for-research/self-hosted-runners/blob/35a90e5c79e3a31997e698bd83beb2f668584104/ubuntu-virtualbox/playbook.yml#L20-L25

File: ubuntu-vagrant/playbook.yml [hunk 3]
```diff
@@ -3,13 +3,13 @@
   hosts: vagrants
   become: yes
   vars:
-    - runner_user: runner
+    - runner_user: tester
```
Comment (jspaaks) [inline ubuntu-vagrant/playbook.yml:6]: same as before 
https://github.com/ci-for-research/self-hosted-runners/blob/35a90e5c79e3a31997e698bd83beb2f668584104/ubuntu-virtualbox/playbook.yml#L20-L25

File: ubuntu-virtualbox/README.md [hunk 4]
```diff
@@ -248,30 +248,30 @@ journalctl -u actions.runner.*
 ### Start the runner each time the machine boots
 
 ```shell
-ansible-playbook playbook.yml --tags enable
+ansible-playbook --ask-become-pass playbook.yml --tags enable
```
Comment (jspaaks) [inline ubuntu-virtualbox/README.md:251]: Not all tasks require root, do you still want to ask the user for root pwd?

File: ubuntu-vagrant/playbook.yml [hunk 5]
```diff
@@ -2,14 +2,12 @@
...
   tasks:
     - name: User for runner service
       user:
-        name: '{{ runner_user }}'
+        name: '{{ ansible_user }}'
         state: present
-    - name: Passwordless sudo for {{ runner_user }} user 
+    - name: Passwordless sudo for {{ runner_user }} user
       lineinfile:
         path: /etc/sudoers.d/{{ runner_user }}
```
Comment (jspaaks) [inline ubuntu-vagrant/playbook.yml]:
```suggestion
        path: /etc/sudoers.d/{{ ansible_user }}
```

File: ubuntu-vagrant/playbook.yml [hunk 6]
```diff
@@ -2,14 +2,12 @@
...
         line: '{{ runner_user }} ALL=(ALL) NOPASSWD: ALL'
```
Comment (jspaaks) [inline ubuntu-vagrant/playbook.yml]:
```suggestion
        line: '{{ ansible_user }} ALL=(ALL) NOPASSWD: ALL'
```

File: windows-vagrant/playbook.yml [hunk 7]
```diff
@@ -2,14 +2,12 @@
...
-  vars:
-    - runner_user: runner
   tasks:
     - name: Runner User
       user:
-        name: '{{ runner_user }}'
+        name: '{{ ansible_user }}'
         state: present
-    - name: Passwordless sudo for {{ runner_user }} user 
+    - name: Passwordless sudo for {{ runner_user }} user
```
Comment (jspaaks) [inline windows-vagrant/playbook.yml]:
```suggestion
    - name: Passwordless sudo for {{ ansible_user }} user
```

File: windows-vagrant/playbook.yml [hunk 8]
```diff
@@ -2,14 +2,12 @@
...
       lineinfile:
         path: /etc/sudoers.d/{{ runner_user }}
```
Comment (jspaaks) [inline windows-vagrant/playbook.yml]:
```suggestion
        path: /etc/sudoers.d/{{ ansible_user }}
```

File: windows-vagrant/playbook.yml [hunk 9]
```diff
@@ -2,14 +2,12 @@
...
         line: '{{ runner_user }} ALL=(ALL) NOPASSWD: ALL'
```
Comment (jspaaks) [inline windows-vagrant/playbook.yml]:
```suggestion
        line: '{{ ansible_user }} ALL=(ALL) NOPASSWD: ALL'
```

File: ubuntu-vagrant/Vagrantfile (hunk 1 above)
Comment (fdiblen) [reply ubuntu-vagrant/Vagrantfile]: reverted the changes

**Pull request merged**